UiPath.FreeRdpClient/UiPath.FreeRdpClient/ServiceRegistryExtensions.cs

```diff
@@ -10,7 +8,12 @@ public static IServiceCollection AddFreeRdp(this IServiceCollection services, st
     {
         Logging.ScopeName = scopeName;
         return services
-            .AddSingleton<IFreeRdpClient, FreeRdpClient>()
-            .AddHostedService<Logging>();
+            .AddSingleton<Logging>()
+            .AddSingleton<IFreeRdpClient>(sp =>
+            {
+                EnsureNativeLogsForwarding();
+                return ActivatorUtilities.CreateInstance<FreeRdpClient>(sp);
+                void EnsureNativeLogsForwarding() => sp.GetRequiredService<Logging>();
+            });
```

Comment:
_nice to have, later_: This could all be simpler:

```csharp
.AddSingleton<Logging>()
.AddSingleton<IFreeRdpClient, FreeRdpClient>()
```

while `FreeRdpClient` would have to:
```csharp
    public FreeRdpClient(ILogger<FreeRdpClient> logger, IServiceProvider serviceProvider)
    {
        _log = logger;
        _ = serviceProvider.GetRequiredService<Logging>(); // ensure Logging was instantiated
    }
```

Also, `Logging` itself could be renamed to `NativeLoggingForwarder`.